tdrs-backend/manifest.yml

```diff
@@ -6,3 +6,15 @@ applications:
   disk_quota: 2G
   docker:
     image: ((docker-backend))
+  env:
+    AV_SCAN_URL: http://((cf-space))-clamav-rest.apps.internal:9000/scan
+- name: clamav-rest
+  instances: 1
+  memory: 1.5G
```

Comment:
Will need to update this to `2G` once the space quota has been increased for the dev space in Cloud.gov.